### PR TITLE
Fix address of the L1 Block Attributes contracts

### PR DESCRIPTION
**Pull request by maurelian**
```diff
- 0x4242424242424242424242424242424242424242
+ 0x4200000000000000000000000000000000000015
```

### CODE REVIEW

**Review by mslipper — APPROVED** [review]

File: opnode/test/system_test.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func TestSystemE2E(t *testing.T) {
 		},
 		cliqueSigners:           []string{"m/44'/60'/0'/0/0"},
 		depositContractAddress:  "0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001",
-		l1InforPredeployAddress: "0x4242424242424242424242424242424242424242",
+		l1InforPredeployAddress: "0x4200000000000000000000000000000000000015",
```
Comment (protolambda) [inline opnode/test/system_test.go:101]: Infor? typo in setup type definition?

**Review by protolambda — COMMENTED** [review]
looks good, but there's a typo in existing related test type, might as well fix that

File: opnode/test/system_test.go [hunk 2]
```diff
@@ -96,9 +95,9 @@ func TestSystemE2E(t *testing.T) {
 			l2OutputHDPath:     10000000,
 			bssHDPath:          10000000,
 		},
-		cliqueSigners:           []string{"m/44'/60'/0'/0/0"},
-		depositContractAddress:  "0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001",
-		l1InforPredeployAddress: "0x4242424242424242424242424242424242424242",
```
Comment (tynes) [inline opnode/test/system_test.go:101]: We should be importing the constants here

**Review by tynes — CHANGES_REQUESTED** [review]
We need to be using the constants where we can so this doesn't happen again

**Review by tynes — APPROVED** [review]

**Pull request merged**